### PR TITLE
fix Issue 14695 - [dlang.org] std.uuid Is Not Listed On The Index Page

### PR DESCRIPTION
**Pull request by CyberShadow**
https://github.com/D-Programming-Language/phobos/pull/3607 follow-up

Made sure I didn't miss anything else this time.

https://issues.dlang.org/show_bug.cgi?id=14695

### CODE REVIEW

**Comment by quickfur** [comment]
Cf. https://github.com/D-Programming-Language/phobos/pull/3611

File: index.d [hunk 1]
```diff
@@ -437,6 +439,19 @@ $(COMMENT
              Experimental modules.
         )
     )
+    $(TR
+        $(TDNW
+            $(LINK2 std_algorithm_comparison.html, std.algorithm.comparison)$(BR)
+            $(LINK2 std_algorithm_iteration.html, std.algorithm.iteration)$(BR)
```
Comment (DmitryOlshansky) [inline index.d]: BR? Shouldn't it be separate td-s?

Comment (quickfur) [inline index.d]: This PR really should be combined with #3611 that addresses this issue.

**Comment by CyberShadow** [comment]
Closing because #3611 incorporates this fix.

**Pull request closed without merging**